packages/mc-dev-authentication/src/create-mc-dev-authentication-middleware.ts

```diff
@@ -39,37 +30,21 @@ function createMcDevAuthenticationMiddleware(
     if (applicationConfig.env.__DEVELOPMENT__?.oidc?.authorizeUrl) {
       // Handle login page for OIDC workflow when developing against a local MC API.
       if (
-        applicationConfig.env.__DEVELOPMENT__?.oidc?.authorizeUrl.startsWith(
+        applicationConfig.env.__DEVELOPMENT__.oidc.authorizeUrl.startsWith(
           'http://localhost'
         )
       ) {
         if (request.originalUrl?.startsWith('/login/authorize')) {
-          if (isDevAuthenticationMiddlewareDisabled) {
-            next();
-          } else {
-            response.end(htmlLogin);
+          if (!isDevAuthenticationMiddlewareDisabled) {
+            // Redirect to the MC API to initiate the authorize flow.
+            const redirectTo = new URL(
+              request.originalUrl,
+              trimTrailingSlash(applicationConfig.env.mcApiUrl)
+            );
+            response.writeHead(301, { Location: redirectTo.toString() }).end();
```

Comment:
This is the main change, we simply redirect to the `/login/authorize` endpoint of the MC API.